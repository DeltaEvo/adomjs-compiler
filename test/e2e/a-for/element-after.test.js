const compileAndCreate = require('../compileAndCreate');

const element = compileAndCreate(`
<template a-for="elem of list">
	<span>{{ elem }}</span>
</template>
<span>After</span>
`);

it('render', () => {
	element.setState({
		list: ['Hello', 'World']
	});

	expect(element.innerHTML).toMatchSnapshot();
});

it('remove elements', () => {
	element.setState({
		list: ['Hello']
	});

	expect(element.innerHTML).toMatchSnapshot();
});
