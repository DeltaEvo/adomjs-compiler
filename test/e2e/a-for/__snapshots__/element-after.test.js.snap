// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`remove elements 1`] = `

<span>
  Hello
</span>
<span>
  After
</span>

`;

exports[`render 1`] = `

<span>
  Hello
</span>
<span>
  World
</span>
<span>
  After
</span>

`;
